src/partisan_client_server_peer_service_manager.erl

```diff
@@ -249,17 +213,6 @@ handle_cast(Msg, State) ->
     lager:warning("Unhandled messages: ~p", [Msg]),
     {noreply, State}.
 
-%% @private
--spec handle_info(term(), state_t()) -> {noreply, state_t()}.
-handle_info(gossip, #state{pending=Pending,
-                           tag=Tag,
-                           membership=Membership,
-                           connections=Connections0}=State) ->
-    Connections = establish_connections(Pending, Membership, Connections0),
```

Comment:
After a connection is formed on line 300, it calls do_gossip to establish a connection with that peer: you've removed this call, so even though membership achieves convergence, you haven't connected to any of the nodes.